<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>server_fileClass</class>
 <widget class="QWidget" name="server_fileClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>server_file</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>481</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_show">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>481</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_listen">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>绑定</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>服务器ip:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_serverip">
      <property name="text">
       <string>192.168.137.1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>服务器port:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_serverport">
      <property name="text">
       <string>9000</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="server_file.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_listen</sender>
   <signal>clicked()</signal>
   <receiver>server_fileClass</receiver>
   <slot>btn_bind_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>-51</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_bind_button_clicked()</slot>
 </slots>
</ui>
